strings/credits.yml

```
@@ -5,1242 +5,1420 @@ Nothing:
-  Sick Kid: "nothing-having kid"
-  Flute Spot: "empty boy does nothing again"
-  Link's Uncle: "your uncle does nothing"
+  King Zora: "mad friend for sale"
+  Potion Shop: "insects for trade"
+  Master Sword Pedestal: "and the mad friend"
+  Sick Kid: "the bug-caught kid"
+  Flute Spot: "bottle boy has mad bee again"
+  Link's Uncle: "your uncle likes arthropods"
+  Blacksmith: "the beekeepers"
 Bombos:
   King Zora: "swirly coin for sale"
   Potion Shop: "shrooms for swirly-coin"
   Master Sword Pedestal: "and the swirly coin"
   Sick Kid: "coin-collecting kid"
   Flute Spot: "medallion boy melts room again"
   Link's Uncle: "your uncle collects coins"
+  Blacksmith: "the mint"
 Ether:
   King Zora: "bolt coin for sale"
   Potion Shop: "shrooms for bolt-coin"
   Master Sword Pedestal: "and the bolt coin"
   Sick Kid: "coin-collecting kid"
   Flute Spot: "medallion boy sees floor again"
   Link's Uncle: "your uncle collects coins"
+  Blacksmith: "the mint"
 Quake:
   King Zora: "wavy coin for sale"
   Potion Shop: "shrooms for wavy-coin"
   Master Sword Pedestal: "and the wavy coin"
   Sick Kid: "coin-collecting kid"
   Flute Spot: "medallion boy shakes dirt again"
   Link's Uncle: "your uncle collects coins"
+  Blacksmith: "the mint"
 Lamp:
   King Zora: "candle for sale"
   Potion Shop: "fungus for illumination"
   Master Sword Pedestal: "and the flashlight"
   Sick Kid: "light-shining kid"
   Flute Spot: "illuminated boy can see again"
   Link's Uncle: "your uncle has night vision"
+  Blacksmith: "the candlemakers"
 Shovel:
   King Zora: "dirt spade for sale"
   Potion Shop: "can you dig it"
   Master Sword Pedestal: "and the flute scoop"
   Sick Kid: "archaeologist kid"
   Flute Spot: "shovel boy digs again"
   Link's Uncle: "your uncle digs it"
+  Blacksmith: "diggy diggy hole"
 OcarinaInactive:
   King Zora: "duck call for sale"
   Potion Shop: "duck-calls for trade"
   Master Sword Pedestal: "and the duck call"
   Sick Kid: "the duck-call kid"
   Flute Spot: "ocarina boy plays again"
   Link's Uncle: "your uncle trains ducks"
+  Blacksmith: "the clayworkers"
 CaneOfSomaria:
   King Zora: "block stick for sale"
   Potion Shop: "twizzle-stick for trade"
   Master Sword Pedestal: "the walking stick"
   Sick Kid: "the block-making kid"
   Flute Spot: "cane boy makes blocks again"
   Link's Uncle: "your uncle makes blocks"
+  Blacksmith: "the woodcarvers"
 Bottle:
   King Zora: "terrarium for sale"
   Potion Shop: "first taste is not free"
   Master Sword Pedestal: "and the terrarium"
   Sick Kid: "the terrarium kid"
   Flute Spot: "bottle boy has terrarium again"
   Link's Uncle: "your uncle likes turtles"
+  Blacksmith: "the glassblowers"
 PieceOfHeart:
   King Zora: "little love for sale"
   Potion Shop: "fungus for life"
   Master Sword Pedestal: "and the broken heart"
   Sick Kid: "the life-giving kid"
   Flute Spot: "life boy feels some love again"
   Link's Uncle: "your uncle is healthy"
+  Blacksmith: "the dawrven breadsmiths"
```

I don't think you typo'd this intentionally, did you?
```suggestion
  Blacksmith: "the dwarven breadsmiths"
```